### ISSUE TITLE
`pendingchannels` RPC command causes panic

### ISSUE THREAD

**Issue by aakselrod** [lightningnetwork/lnd#144]
Reported by `mably` on Slack, the following panic occurs:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x4276d4]

goroutine 322 [running]:
panic(0xb32100, 0xc04200a090)
        /dev/go/src/runtime/panic.go:500 +0x1af
main.(*rpcServer).PendingChannels(0xc04240fbf0, 0x1bf9140, 0xc0425890e0, 0xc042716f5c, 0x0, 0xc0422d7ac8, 0x6147c3)
        /dev/go/src/github.com/lightningnetwork/lnd/rpcserver.go:752 +0x2b4
github.com/lightningnetwork/lnd/lnrpc._Lightning_PendingChannels_Handler(0xbf9d00, 0xc04240fbf0, 0x1bf9140, 0xc0425890e0, 0xc04226dda0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /dev/go/src/github.com/lightningnetwork/lnd/lnrpc/rpc.pb.go:3113 +0x287
github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc0422000c0, 0x10d26e0, 0xc0422325b0, 0xc042000c00, 0xc0424d6660, 0x10f27f0, 0xc0425891a0, 0x0, 0x0)
        /dev/go/src/github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc/server.go:708 +0xbdf
github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc.(*Server).handleStream(0xc0422000c0, 0x10d26e0, 0xc0422325b0, 0xc042000c00, 0xc0425891a0)
        /dev/go/src/github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc/server.go:892 +0x6ab
github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc0426dfaa0, 0xc0422000c0, 0x10d26e0, 0xc0422325b0, 0xc042000c00)
        /dev/go/src/github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc/server.go:468 +0xb2
created by github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
        /dev/go/src/github.com/lightningnetwork/lnd/vendor/google.golang.org/grpc/server.go:469 +0xaa
```
This happens when the `pendingchannels` RPC handler attempts to dereference the channel point in one of the pending channels. The channel was opened directly after the peer force-closed all existing channels. Restarting the local `lnd` process and attempting again resulted in another panic. Restarting the peer `lnd` process and starting the local `lnd` process and attempting again resulted in success.

**Comment by spolu** [comment]
ooc where is "Slack"? is there a public (private?) slack room one can join?

**Comment by AndrewSamokhvalov** [comment]
@spolu https://bitcoincore.slack.com/messages/ln-testers/

**Comment by spolu** [comment]
🙏

**Comment by spolu** [comment]
@andrewsamokhvalov Is that a private channel? I'm spolu on bitcoincore.org slack.

**Comment by AndrewSamokhvalov** [comment]
@aakselrod As far I understand this happens because `FundingOutpoint` of reservation initialized only on stage of receiving `SingleFundingComplete` message.

```
// handleSingleFunderSigs is called once the remote peer who initiated the
// single funder workflow has assembled the funding transaction, and generated
// a signature for our version of the commitment transaction. This method
// progresses the workflow by generating a signature for the remote peer's
// version of the commitment transaction.
func (l *LightningWallet) handleSingleFunderSigs(req *addSingleFunderSigsMsg) {
       . . .
	pendingReservation.partialState.FundingOutpoint = req.fundingOutpoint
       . . .
```

**Comment by Roasbeef** [comment]
Fixed by #145
